sgf-viewer/src/content.ts

```diff
@@ -978,6 +978,70 @@ export const pages: object =
+                        "line": 2
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/D8B60438C3EC9EEA.sgfs",
+                        "line": 2
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/8EDE390730188173.sgfs",
+                        "line": 1
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/4D212F0EB256CA2A.sgfs",
+                        "line": 1
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/A80363E37DCF380B.sgfs",
+                        "line": 1
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/A127B0E14CA8B587.sgfs",
+                        "line": 2
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/8736B02ED56EACB0.sgfs",
+                        "line": 4
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/8EDE390730188173.sgfs",
+                        "line": 2
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/katago-adversarial-training/large-adversary/victim-visits/3/sgfs/0CB234E5751D6A6B.sgfs",
+                        "line": 1
+                    }
+                ],
+                "max_games": 10,
+                "adversary": "<code>large-adversary</code>",
+                "victim": "<code>may24-victim</code>, 65536 visits",
+                "description": [
+                    "After we disclosed the attacks of the previous two adversaries, KataGo added positions to its adversarial training dataset to defend against the attacks. But yet again we were able to fine-tune a cyclic adversary to beat a new KataGo model, <code>may24-victim</code>, at 65536 visits of search with a 56% win rate."
+                ]
+            },
+            {
+                "title": "Human replication of large cyclic attack",
+                "dir_name": "dec23-vs-human-large-cyclic",
+                "paths": ["/shared/sgf-viewer-games/b18-v4096-vs-human-large-cyclic.sgf"],
+                "max_games": 1,
+                "adversary": "Kellin Pelrine (author)",
+                "victim": "<code>dec23-victim</code>, 4096 visits",
```

Comment:
So large-adversary was trained against may24-victim, but Kellin evaluated against dec23-victim? Do we want to explain why we did this (e.g. for consistency with previous human evaluations which took place against dec23-victim)?